x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/BertJapaneseTokenizerTests.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.ml.inference.nlp.tokenizers;
+
+import org.apache.lucene.analysis.Tokenizer;
+import org.apache.lucene.analysis.ja.JapaneseTokenizer;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.core.ml.inference.trainedmodel.BertTokenization;
+import org.elasticsearch.xpack.core.ml.inference.trainedmodel.Tokenization;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+
+public class BertJapaneseTokenizerTests extends ESTestCase {
+
+    public static final List<String> TEST_CASED_VOCAB = List.of(
+        "日本",
+        "語",
+        "使",
+        "テスト",
+        "、",
+        "。",
+        "Bert",
+        "##Japanese",
+        "##Tokenizer",
+        "Elastic",
+        "##search",
+        BertTokenizer.CLASS_TOKEN,
+        BertTokenizer.SEPARATOR_TOKEN,
+        BertTokenizer.MASK_TOKEN,
+        BertTokenizer.UNKNOWN_TOKEN,
+        BertTokenizer.PAD_TOKEN
+    );
+
+    private List<String> tokenStrings(List<? extends DelimitedToken> tokens) {
+        return tokens.stream().map(DelimitedToken::toString).collect(Collectors.toList());
+    }
+
+    public void testTokenize() {
+        try (
+            BertTokenizer tokenizer = BertJapaneseTokenizer.builder(
+                TEST_CASED_VOCAB,
+                new BertTokenization(null, false, null, Tokenization.Truncate.NONE, -1)
+            ).build()
+        ) {
+
+            String msg = "日本語で、ElasticsearchのBertJapaneseTokenizerを使うテスト。";
+            // List<TokenizationResult.Tokens> tokens = tokenizer.tokenize(msg, Tokenization.Truncate.NONE, 1, 0);
+            // System.out.println(List.of(tokens));
```

Comment:
nit: please delete the commented out code.

Reply:
Deleted